trunk/SUAVE/Methods/Aerodynamics/Common/Higher_Fidelity/Drag/untrimmed.py

```diff
@@ -1,12 +1,14 @@
-## @ingroup Methods-Aerodynamics-SU2_Euler
+## @ingroup Methods-Aerodynamics-Higher_Fidelity-Drag
```

Comment:
Do a compare of the texts. They are different enough to justify separate scripts.